install/kubernetes/templates/istio-pilot.yaml.tmpl

```diff
@@ -1,3 +1,49 @@
+################################
+# Pilot
+################################
+# Pilot CRDs
```

Comment:
how do we know there are no typos / won't be typos in the future ? does some test fail ? some error in logs ?

Reply:
Routing rules would fail @kyessenov may have more details.